service_tools/scripts/service_relay

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
+import rospy
+import rosservice
+
+if __name__ == "__main__":
+    argv = rospy.myargv()
+    if len(argv) != 3:
+        rospy.logerr("service_relay IN OUT")
+
+    rospy.init_node("service_relay")
+
+    local_name, remote_name = argv[1], argv[2]
+
+    rospy.wait_for_service(remote_name)
```

Comment:
For our use case we don't have a default timeout. (It depends on someone to press the init button..)